packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/ccn/1486/steps/Salaire.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { Field } from "react-final-form";
+import { FieldArray } from "react-final-form-arrays";
+import styled from "styled-components";
+import { subMonths, format } from "date-fns";
+import frLocale from "date-fns/locale/fr";
+import { Table as UITable, theme, Toast } from "@cdt/ui";
+
+import { Input, InlineError } from "../../../../common/stepStyles";
+import { isNumber } from "../../../../common/validators";
+import { YesNoQuestion } from "../../../components/YesNoQuestion";
+
+export const Salaire = ({ form }) => {
+  const data = form.getState().values;
+  return (
+    <>
+      <Toast type="info">
+        Le salaire à renseigner pour cette convention collective inclut:
+        <ul>
+          <li>les primes prévues au contrat de travail</li>
+        </ul>
+        <strong>mais elle exclut</strong>:
+        <ul>
+          <li>
+            les heures supplémentaires au delà de l’heure normale de
+            l’entreprise
+          </li>
+          <li>
+            les majorations/indemnités liées à un déplacement ou un détachement
+          </li>
+        </ul>
+      </Toast>
+      <StyledToast variant="warning">
+        Attention, les éventuelles <strong>primes</strong> renseignées
+        précédemment ne seront pas prises en compte dans le calcul de
+        l’indemnité de votre convention collective. Si vous en avez perçues,
+        merci de les intégrer à votre rémunération ci-après.
+        <br />
+        Aussi, si il y en a, les mois effectués en{" "}
+        <strong>temps partiel</strong> sont à considérer comme ceux effectués en
+        temps plein.
+      </StyledToast>
+      <YesNoQuestion
+        name="hasBrancheNewSalaire"
+        label="De ce fait, avez-vous une modification à faire dans la déclaration de votre salaire&nbsp;?"
+      />
+      {data.hasBrancheNewSalaire && <NewSalaire form={form} data={data} />}
+    </>
+  );
+};
+
+const NewSalaire = ({ form, data }) => (
```

Comment:
I would merge NewSalaire into the component above
to improve clarity I would rather extract notice / toast into another component.
that way you don't need to you pass down form and data